webapp/src/js/bootstrapper/index.js

```diff
@@ -87,6 +87,17 @@
 
   /* pouch db set up function */
   module.exports = (POUCHDB_OPTIONS) => {
+    utils.checkApiAccessible()
```

Comment:
Also, this isn't blocking on the check so I'm not sure how it would work. I think you need to return or await the promise resolution.